app/controllers/confirmations_controller.rb

```diff
@@ -6,8 +6,9 @@ def show
 
     if resource.errors.empty?
       set_flash_message!(:notice, :confirmed)
+      sign_in(resource)
+
       if resource.creator?
-        sign_in(resource)
```

Comment:
Ensure users are signed in after email confirmation so registration flow goes smoothly not just for creator email confirmation. Users are redirected to onboarding (see video recording in PR description)